cloudformation/extraction.yaml

```diff
@@ -25,6 +26,14 @@ ExtractFunction:
               - sqs:GetQueueAttributes
               - sqs:ReceiveMessage
             Resource: !GetAtt ExtractQueue.Arn
+          - Effect: Allow
+            Action:
+              - "sns:*"
+            Resource: "*"
```

Comment:
This should be more locked down to the actual resources.

Reply:
Also, this is the function, it does not need SNS access as it is not doing anything with SNS.